frontend/src/scenes/session-recordings/components/replayActiveUsersTableLogic.ts

```diff
@@ -23,18 +23,47 @@ export const replayActiveUsersTableLogic = kea<replayActiveUsersTableLogicType>(
         countedUsers: {
             loadCountedUsers: async (_, breakpoint): Promise<{ person: PersonType; count: number }[]> => {
                 const q = hogql`
-                    select p, any (pp), count () as c
-                    from (
-                        select any (person_id) as p, any (person.properties) as pp
-                        from raw_session_replay_events
-                        where min_first_timestamp <= now()
-                        and min_first_timestamp >= now() - toIntervalDay(7)
-                        group by session_id
-                        having date_diff('second', min (min_first_timestamp), max (max_last_timestamp)) > 5000
-                        ) as q
-                    group by p
-                    order by c desc
-                        limit 10
+                  WITH
+session_distinct_ids AS (
+    SELECT any(distinct_id) AS di
+    FROM raw_session_replay_events
+    WHERE min_first_timestamp <= now()
+      AND min_first_timestamp >= now() - toIntervalDay(7)
+    GROUP BY session_id
+    HAVING date_diff('second', min(min_first_timestamp), max(max_last_timestamp)) > 5
+),
+
+person_ids as (
+  SELECT
+    p.id AS person_id, p.properties as person_properties, p.pdi.distinct_id as the_pdi
```

Comment:
Do you have person ID squashing and PoE on replay events? I would have expected that it would be faster to use PoE and the person ID overrides table, but I'm not sure that this is possible here